cypress/e2e/gui/paginaInicial.cy.js

```diff
@@ -0,0 +1,19 @@
+/// <reference types="cypress" />
+
+describe('Página inicial - Saudação', () => {
+    it('Saudação Padrão', () => {
+        cy.visit('http://localhost:3000/')
+
+        cy.get('[data-testid="table"] > :nth-child(1)')
+            .should('contain', 'Hi there! It is now')
+    });
+    it('Saudação Personalizada', () => {
+        cy.visit('http://localhost:3000/')
+
+        cy.get('[data-testid="name"]')
+            .type('Eduarda')
+
+        cy.get('[data-testid="table"] > :nth-child(1)')
```

Comment:
E aqui poderia ser o seguinte: `cy.contains('p', 'Hi Eduarda!').should('be.visible')`

> Ou seja, busque um parágrafo (uma tag `p`) a qual possua o conteúdo `Hi Eduarda!` e garanta que tal parágrafo está visível.

Reply:
obrigada pela dica